modules/wsl-conf.nix

```
@@ -0,0 +1,90 @@
+{ lib, pkgs, config, ... }:
+with builtins; with lib; {
+
```

```suggestion

with lib; {
```
almost all builtins are inherited into lib.

---

Didn't know that works - thanks